src/routes/admin/profiles/create.test.ts

```diff
@@ -31,6 +33,44 @@ describe('Profiles - Create', () => {
     })
     expect(resSpy.status).toHaveBeenCalledWith(201)
   })
+  it('should create when generate random password true for acm', async () => {
```

Comment:
test descriptor doesn't match. generate random password is false?

Reply:
Thank you. Its updated